test/parallel/test-fs-options-immutable.js

```diff
@@ -58,9 +58,7 @@ if (common.canCreateSymLink()) {
   );
 }
 
-if (!common.isAix) {
-  // TODO(thefourtheye) Remove this guard once
-  // https://github.com/nodejs/node/issues/5085 is fixed
+{
```

Comment:
Can we just get rid of this start brace and its matching end brace and re-indent the enclosed code?